test/models/settings_model_test.dart

```diff
@@ -17,6 +17,7 @@ void main() {
     'theme': 1,
     'nrOfDaysToDisplay': 7,
     'greyScale': false,
+    'completedActivityOption': 2,
```

Comment:
This should probably also be checked in some expect statement in the tests. Similar to how it is done on lines 35-53.